lib/conjur/command.rb

```diff
@@ -100,6 +110,30 @@ def command_impl_for_list(global_options, options, args)
         end
       end
       
+      def validate_privileges message, &block
+        valid = begin
+          yield
+        rescue RestClient::Exception
```

Comment:
Good catch
On Thu, May 14, 2015 at 5:22 PM apotterri <notifications@github.com> wrote:

> In lib/conjur/command.rb
> <https://github.com/conjurinc/cli-ruby/pull/108#discussion_r30364526>:
>
> > @@ -100,6 +110,30 @@ def command_impl_for_list(global_options, options, args)
> >          end
> >        end
> >
> > +      def validate_privileges message, &block
> > +        valid = begin
> > +          yield
> > +        rescue RestClient::Exception
>
> Should this check the status on the response from the API, too? If I make
> a typo in my resource name, I now get "You can't administer this record",
> rather than "404 Resource Not Found" (which seems more helpful).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conjurinc/cli-ruby/pull/108/files#r30364526>.
>